### PR TITLE
Windows build fixes

### PR DESCRIPTION
**Pull request by jamesjrg**
Maybe this will fix the Windows build issues

### CODE REVIEW

**Comment by haf** [comment]
Let's see what AppVeyor says :)

Thanks for looking into the build issues on Windows for us!

File: examples/Fibonacci/Fibonacci.fsproj [hunk 1]
```diff
@@ -32,18 +32,17 @@
     <Externalconsole>true</Externalconsole>
     <Tailcalls>true</Tailcalls>
   </PropertyGroup>
+  <Import Project="$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.1\Framework\v4.0\Microsoft.FSharp.Targets" />
```
Comment (haf) [inline examples/Fibonacci/Fibonacci.fsproj]: Similarly conditionally compile this?

File: src/Suave.Testing/Suave.Testing.fsproj [hunk 2]
```diff
@@ -36,27 +36,7 @@
...
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Numerics" />
-    <Reference Include="System.Net.Http" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Testing.fs" />
-    <Compile Include="AssemblyVersionInfo.fs" Condition="Exists('AssemblyVersionInfo.fs')" />
-  </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.1\Framework\v4.0\Microsoft.FSharp.Targets" />
```
Comment (haf) [inline src/Suave.Testing/Suave.Testing.fsproj]: Conditionally compile?

File: .gitignore [hunk 3]
```diff
@@ -16,4 +16,5 @@ AssemblyVersionInfo.fs
 miTLS/
 Backup/
 paket.exe
-!paket.lock
\ No newline at end of file
+!paket.lock
+**.suo
```
Comment (haf) [inline .gitignore]: One star should be enough

Comment (jamesjrg) [inline .gitignore]: a single star doesn't check subdirectories, and the file will usually be in /src/

**Pull request closed without merging**